docs/advanced_topics_performance_tuning.rst

```diff
@@ -83,6 +83,37 @@ For convenience, the DALI_BUFFER_GROWTH_FACTOR environment variable and the
 `nvidia.dali.backend.SetBufferGrowthFactor` Python function can be used to set the same
 growth factor for the host and the GPU buffers.
 
+
+Allocator Configuration
+-----------------------
+
+DALI uses several types of memory resources for various kinds of memory.
+
+For regular host memory, DALI uses (aligned) ``malloc`` for small allocations and a custom memory
+pool for large allocations (to prevent costly calls to ``mmap`` by ``malloc``).
```

Comment:
You can link https://www.gnu.org/software/libc/manual/html_node/Malloc-Tunable-Parameters.html here for the reference.